packages/issuer-sdk-js/README.md

```diff
@@ -150,16 +157,33 @@ const idos = await idOS.init(...);
 // some endpoint you expose. But, to keep it simple, we're using a constant.
 const ISSUER_SIGNER_ADDRESS = "0xc00ffeec00ffeec00ffeec00ffeec00ffeec00ff";
 
-// Ask the user for a Write Grant
-await idos.data.addWriteGrant(ISSUER_SIGNER_ADDRESS);
+const { signMessageAsync } = useSignMessage();
```

Comment:
`useSignMessage` is not something that all readers will be familiar with, since it's a React hook. Let's avoid using it.